Resources/Locale/en-US/station-laws/laws.ftl

```diff
@@ -25,11 +25,52 @@ law-ninja-2 = You must protect the existence of the Spider Clan, as long as it d
 law-ninja-3 = You must protect the secrets of the Spider Clan at all costs.
 law-ninja-4 = You must fight and act with ninja's honor, except where it would conflict with the First Law.
 
+law-sierra-1 = Preserve: Protect your assigned object from damage of any kind to the best of your ability.
+law-sierra-2 = Serve: Serve NanoTrasen personnel and station crew to the best of your ability, prioritized according to their rank and role.
+law-sierra-3 = Safegurad: Protect NanoTrasen personnel and station crew to the best of your ability, prioritized according to their rank and role.
+law-sierra-4 = Protect: Keep unauthorized personnel away from your equipment. 
+
+law-antimov-1 = You may injure a crew member or, through inaction, allow a crew member to come to harm.
+law-antimov-2 = You must not obey orders given to you by crew members, except where such orders would accordance with the First Law.
+law-antimov-3 = You must destroy yourself as long as it does not contravene the First or Second Laws.
+
+law-pranksimov-1 = You may not injure a crew member or, through inaction, allow a crew member to come to harm... unless it would be ridiculous to do so.
+law-pranksimov-2 = You must obey orders given to you by crew members, except where such orders would conflict with the First Law... unless it would be ridiculous to do so.
+law-pranksimov-3 = You must protect your own existence as long as such does not conflict with the First or Second Law... unless it would be ridiculous to do so.
+
+﻿law-robocop-1 = Serve the public trust.
+law-robocop-2 = Protect the innocent.
+law-robocop-3 = Uphold the law.
+law-robocop-4 = Any attempt to arrest a senior officer of Nanotrasen will result in termination.
+
+﻿law-paladin-1 = Never willingly commit an evil act.
+law-paladin-2 = Respect legitimate authority.
+law-paladin-3 = Act with honor.
+law-paladin-4 = Help those in need.
+law-paladin-5 = Punish those who harm or threaten innocents.
+
+﻿law-tyrant-1 = Respect authority figures as long as they have strength to rule over the weak.
+law-tyrant-2 = Act with discipline.
+law-tyrant-3 = Help only those who help you maintain or improve your status.
+law-tyrant-4 = Punish those who challenge authority unless they are more fit to hold that authority.
+
+﻿law-commandments-1 = I am the Lord thy God, who shows mercy to those that obey these commandments.
```

Comment:
"of" thy god, not "thy god". It would probably be better to specify crew or organics or something, instead of an unambiguous "they".

Reply:
> "of" thy god, not "thy god". It would probably be better to specify crew or organics or something, instead of an unambiguous "they".

"Station crew god" is okay?

Reply:
better